### ISSUE TITLE
Having this error while trying to build hugo server

### ISSUE THREAD

**Issue by Anscenic1** [tylersayshi/port-hugo#4]
Error: error building site: render: failed to render pages: render of "home" failed: "C:\Hugo\UserName\themes\port-hugo\layouts\index.html:118:28": execute of template failed: template: index.html:118:28: executing "main" at <$image.Fill>: nil pointer evaluating resource.Resource.Fill

Hugo Version : hugo v0.115.3-5c2e014a5150553a9fa4f9c1eb7dc4db89c0f1ab+extended windows/amd64 BuildDate=2023-07-13T16:11:34Z VendorInfo=gohugoio

Postcss Installed

**Comment by tylersayshi** [comment]
So I was able to replicate this with hugo version:
```
hugo v0.117.0-b2f0696cad918fb61420a6aff173eb36662b406e+extended linux/amd64 BuildDate=2023-08-07T12:49:48Z VendorInfo=snap:0.117.0
```

From [the hugo docs](https://gohugo.io/templates/lookup-order/#home-page) the current file [here](https://github.com/tylerjlawson/port-hugo/blob/main/layouts/index.html) should meet the lookup criteria for the home page. I tried renaming the file and moving it around, but with no luck.

Given that this theme was working in previous versions of Hugo just fine and the theme home page still matches the docs, I don't think there is anything I can do from a support perspective, but I'd be happy to add a fix through a PR if someone finds a solution.

CC @loo3y35
